includes/sdk/FreemiusWordPress.php

```diff
@@ -354,8 +354,11 @@ static function RemoteRequest( $pUrl, $pWPRemoteArgs ) {
             $response = wp_remote_request( $pUrl, $pWPRemoteArgs );
 
             if (
-                empty( $response['headers'] ) ||
-                empty( $response['headers']['x-api-server'] )
+                is_array( $response ) &&

```

Comment:
@fajardoleo The `wp_remote_request` is guaranteed to return either an `array` or `WP_Error`. This is what we are doing in this method (+ the replica method in `class-fs-api.php`) too.

So I propose to just enhance the `@return` of this function and also `FS_Api#remote_request` to `@return array|WP_Error`.

Now about this `is_array` check, I was thinking if `!is_wp_error( $response )` is more semantic here. Please take a 2nd look.

Reply:
@swashata Thanks! The PHP documentation comment has been updated.